docker/docker-compose.yml

```diff
@@ -15,11 +15,12 @@ services:
 
   wis2node-ui:
     container_name: wis2node-ui
-    build: https://github.com/wmo-im/wis2node-ui.git#main
+    image: webbben/wis2node-ui
```

Comment:
Shouldn't we point to the canonical repo?

Reply:
Yes they should. The builds can't be run from Github Repositories as you can see [from this error](https://github.com/webb-ben/wis2node/actions/runs/1584965243) in the workflow. Current workaround is Docker Image variants of these that I have been keeping up to date with their canonical repo.
![image](https://user-images.githubusercontent.com/40066515/148101031-f0cc4ce2-e69e-487f-a5fe-33b65e36ef86.png)